controller/telemetry/server.go

```diff
@@ -28,6 +27,10 @@ import (
 	k8sV1 "k8s.io/api/core/v1"
 )
 
+const (
+	requestsMetric = "reports_total"
```

Comment:
I recommend calling this const `reportsMetric`, for consistency with `reportsLabels` and `reportsTotal` (and to avoid confusion with `requestsLabels` and `requestsTotal`).

Reply:
Arg, definitely meant to type `reportsMetrics`.  thanks for catching